doc/nrf/test_and_optimize.rst

```diff
@@ -1,17 +1,165 @@
+      #. Power on the DK.
+      #. Click :guilabel:`Select Device` and select the particular kit entry from the drop-down list in the Serial Terminal.
+      #. Select the serial port you want to connect to from the drop-down list.
+      #. Configure the serial port connection parameters.
+         See the `Serial Terminal configuration`_ page for more information.
+      #. Click :guilabel:`Connect to port`.
+
+      nRF Connect Serial Terminal starts the serial communication with the kit.
+
+      For more information about how to connect using the app, see the `Serial Terminal documentation <Connecting using Serial Terminal_>`_.
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      The |nRFVSC| includes an integrated serial port and RTT terminal, which you can use to connect to your board.
+      For detailed instructions, see `How to connect to the terminal`_ on the `nRF Connect for Visual Studio Code`_ documentation site.
+
+      The extension is available for Windows, Linux, and macOS.
+
+   .. group-tab:: PuTTY (Windows only)
+
+      PuTTY is available only for Windows.
+
+      To connect to DK with PuTTY using CDC-UART, complete the following steps:
+
+      1. Connect the DK to the computer with a USB cable, and then turn on the DK.
+
+         **LED1** starts blinking.
+
+      #. Open PuTTY.
+      #. Select **Serial** as the **Connection type** under **Basic options for your PuTTY session**.
+
+         The text fields above the selection change to **Serial line** and **Speed**.
+
+      #. Click the **Terminal** category in the category selection tree to see options controlling the terminal.
+      #. Enable the following options:
+
+         * Implicit CR in every LF
+         * Implicit LF in every CR
+         * Local echo: Force on
+         * Local line editing: Force on
+
+         .. figure:: images/putty.svg
+            :alt: PuTTY configuration for sending commands through UART
+
+            PuTTY configuration for sending commands through UART
+
+      #. Click the **Serial** category under the **Connection** category in the category selection tree to see options controlling the local serial line.
+      #. Type the COM port corresponding to your DK in the **Serial line to connect to** field.
+
+         Depending on what devices you have connected to your computer, you might have several choices.
+         To find the correct port:
+
+         a. Right-click on the Windows Start menu, and select **Device Manager**.
+         #. In the **Device Manager** window, scroll down and expand **Ports (COM & LPT)**.
+         #. Find the port named *JLink CDC UART Port* and note down the number in parentheses.
+
+            If you have more than one J-Link UART Port, unplug the one that you want to use, plug it back in, and observe which one appeared last.
+
+            Your DK can show up as two consecutive COM ports.
+            If this is the case, you need to test which COM port is the correct one.
```

Comment:
Should this information be also in the Serial Terminal section? Or somewhere outside of the tabs? It doesn't make sense to exist only here.

Reply:
It is already covered to some degree here: https://docs.nordicsemi.com/bundle/nrf-connect-serial-terminal/page/selecting_serial_port.html
I'd rather update this in the ST guide and rely on the link to the ST docs.

Reply:
Tasked here https://nordicsemi.atlassian.net/browse/NCD-685